### PR TITLE
Release IIB 6.7.3

### PR DESCRIPTION
**Pull request by chandwanitulsi**

### CODE REVIEW

**Comment by chandwanitulsi** [comment]
@release-engineering/exd-guild-hello-operator please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## Unreleased
 
+## 6.7.3
+- Using ubi8 for iib messaging container image
+- Enable weekly rebuild of IIB images and manual trigger of GH action
+- Upgrade pytest to 7.2.0
+- Increase wait time for grpc calls
+- Remove misleading log from API response
+- Retry decorators replaced by tenacity
+- Upgrade py to 1.11.0
```
Comment (lipoja) [inline CHANGELOG.md:16]: Didn't we removed it?

Comment (lipoja) [reply CHANGELOG.md:16]: By this PR: https://github.com/release-engineering/iib/pull/457/files

Comment (chandwanitulsi) [reply CHANGELOG.md:16]: Yup, you are right, it was removed in later PRs

**Comment by lipoja** [comment]
Closing in favor of v1.8.0 release - https://github.com/release-engineering/iib/pull/468

**Pull request closed without merging**